slicer/SemiologyVisualization.py

```diff
@@ -161,10 +161,11 @@ def getInclusionsWidget(self):
       ' and describes the related seizure semiology, e.g. papers looking at TLE, FLE, OLE, etc.'
     )
     self.seizureSemiologyCheckBox.setToolTip(
-      'When the paper selects a sample of patients based on their siezure semiology'
-      ' (e.g. genital automatisms, nose-wiping, gelastic, ictal kissing),'
-      ' or other factors (specific techniques used, specific associated conditions'
-      ' like FCD etc.), and provides details of epileptogenic zone localisation/lateralisation'
+      'When the paper pre-selects a sample of patients based on their seizure semiology'
+      ' (e.g. nose-wiping, gelastic, ictal kissing),'
+      ' or takes a cohort of unselected patients with epilepsy,'
+      ' or pre-selects based on other non-topological factors (specific techniques or conditions'
+      ' e.g. FCD), and provides details of epileptogenic zone localisation/lateralisation'
```

Comment:
This is pasted from the README, so you might want to change the text there as well.

Reply:
yes readme is updated on this too